api/src/main/java/com/staxpayments/api/models/Merchant.kt

```diff
@@ -0,0 +1,36 @@
+package com.staxpayments.api.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+
+@Serializable
+data class Merchant(
+    val id: String,
+    val mid: String?,
+    val status: String?,
+    val subdomain: String?,
+    val plan: Plan?,
+    val options: JsonElement?,
+    val processor: String?,
+    val branding: Branding?,
```

Comment:
Convert `Branding?` to `JsonElement?`

It won't be guaranteed to have the options you're serializing, so it's safer to keep it as a `JsonElement`